packages/components/src/modal/index.js

```diff
@@ -101,6 +102,11 @@ class Modal extends Component {
 		ariaHelper.showApp();
 	}
 
+	onModalOnly( ev ) {
+		// Stop all onMouseDown events propagating further - they should only go to the modal
```

Comment:
Particularly since all the other functions are documented, seems this could have worked well as a JSDoc block for the function itself to explain its purpose.